src/gpu/sycl/ref_prelu.hpp

```diff
@@ -0,0 +1,225 @@
+/*******************************************************************************
+* Copyright 2022 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#ifndef GPU_SYCL_GENERIC_PRELU_HPP
+#define GPU_SYCL_GENERIC_PRELU_HPP
+
+#include "common/broadcast_strategy.hpp"
+#include "common/dnnl_thread.hpp"
```

Comment:
You don't need host threading utils for GPU

Reply:
Yes, Removed dnnl_thread utils.